src/aat/resources/features/F-1008 - Submit Event Creation Handle Case Links/CreateCases/Citizen/F-1008_Test_Data_Base_Negative_Citizen.td.json

```diff
@@ -0,0 +1,34 @@
+{
+	"_guid_": "F-1008_Test_Data_Base_Negative_Citizen",
+
+	"productName": "CCD Data Store",
+	"operationName": "Submit event creation as Citizen",
+
+	"method": "POST",
+  "uri": "/citizens/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/{cid}/events",
```

Comment:
If a **common TD file** to cover the basics of the API signature does not exist can we create one and use it as this endpoint is being used multiple times in these tests:

* F-1006_Test_Data_Base_Citizen
* F-1008_Test_Data_Base_Citizen
* F-1008_Test_Data_Base_Negative_Citizen

Reply:
Done in [2e98c6c](https://github.com/hmcts/ccd-data-store-api/pull/1719/commits/2e98c6c69d70f77077a6395aad95e21a9135f99a)